layouts/shortcodes/cncf-landscape.html

```diff
@@ -54,6 +54,25 @@
   /* handle initially-set fragment */
   hashChangeHandler();
 });
+
+var documentReady = function (callback) {
+  if (document.readyState !== "loading") callback();
+  else document.addEventListener("DOMContentLoaded", callback);
+};
+
+documentReady(function () {
+  function checkPage() {
+    var trainingPage = "https://landscape.cncf.io/category=kubernetes-training-partner&format=logo-mode&grouping=category&embed=yes";
```

Comment:
This hard-codes a specific category. I don't think any part of this shortcode should hard code category names.
Instead, take those as parameters.